python/protobuf.c

```diff
@@ -96,7 +96,8 @@ PyObject *PyUpb_ObjCache_Get(const void *key) {
 // -----------------------------------------------------------------------------
 
 typedef struct {
-  PyObject_HEAD upb_arena* arena;
+  PyObject_HEAD
```

Comment:
I suspect you can add a // at the end of the line to force clang-format to line break appropriately

Reply:
I tried that, and it does prevent the line join, but it indents the next line:

```c
typedef struct {
  PyObject_HEAD  // Macro.
      upb_arena* arena;
} PyUpb_Arena;
```